server/api/services/user.service.js

```diff
@@ -14,7 +18,41 @@ const checkUsernameExists = async ({ username }) => {
   };
 };
 
+const checkEmailExists = async ({ email }) => {
+  const user = await UserRepository.getByEmail(email);
+  return {
+    emailExists: Boolean(user)
+  };
+};
+
+const sendForgetPasswordEmail = async ({ email }) => {
+  const isExist = await checkEmailExists({ email });
+  if (!isExist.emailExists) {
+    return { failure: 'Email is not exist' };
+  }
+  const user = await UserRepository.getByEmail(email);
+  const token = jwt.sign({ data: user.email }, secret, { expiresIn: '1h' });
```

Comment:
Should be used `server/helpers/token.helper.js`

Reply:
fixed